<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkUSDeviceManagerWidgetControls</class>
 <widget class="QWidget" name="QmitkUSDeviceManagerWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>234</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkUSDeviceManagerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QmitkServiceListWidget" name="m_ConnectedDevices" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="m_BtnRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/USUI.qrc">
         <normaloff>:/USUI/minus.png</normaloff>:/USUI/minus.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="m_BtnNewDevice">
       <property name="text">
        <string>New Video Device</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/USUI.qrc">
         <normaloff>:/USUI/plus.png</normaloff>:/USUI/plus.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="m_BtnEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit Video Device</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="m_BtnActivate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Activate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkServiceListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkServiceListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/USUI.qrc"/>
 </resources>
 <connections/>
</ui>
